lib/matplotlib/ticker.py

```diff
@@ -358,7 +359,16 @@ class StrMethodFormatter(Formatter):
 
     The field used for the tick value must be labeled *x* and the field used
     for the tick position must be labeled *pos*.
+
+    Negative numeric values (e.g., -1) will use a dash, not a Unicode minus;
+    use mathtext to get a Unicode minus by wrapping the format specifier with $
+    (e.g. "${x}$").
```

Comment:
Because it applies both to FormatStrFormatter and to StrMethodFormatter.  (Actually this may change with https://github.com/matplotlib/matplotlib/pull/27602...)